src/components/Schedule/Counter.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+import { Icon } from "../Icon";
+import minusIcon from "../../static/svg/minus.svg";
+import plusIcon from "../../static/svg/plus.svg";
+import { Button } from "../Button";
+
+export const Counter = () => {
+  const [counter, setCounter] = useState(1);
+  const increment = () => setCounter(counter + 1);
+  const decrement = () => setCounter(counter - 1);
+  return (
+    <>
+      <div className="pt-16 text-12 font-lato font-bold">Počet miest</div>
+      <div className="pt-4 flex">
+        <div className="flex justify-between" style={{ width: "168px" }}>
```

Comment:
the same, ask for adding this value to the tailwind, most likely someone else will need it also